NEWS.md

```diff
@@ -1,5 +1,9 @@
 # censored (development version)
 
+* Predictions of survival probabilities for proportional hazards models with 
```

Comment:
no big deal but I usually do news entries on a single line for better diffs

Reply:
oh, good point, thanks! 👍